packages/guui/htmlTemplate.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import resetCSS from './lib/reset-css';
+import { hashedPath, staticPath } from './lib/asset-path';
+
+export default ({
+    title = 'The Guardian',
+    vendor = ['javascript', 'vendor.js'],
+    fonts = ['css', 'fonts.css'],
+    bundle,
+    css,
+    html,
+    data,
+    cssIDs,
+    nonBlockingJS = '',
+}: {
+    title?: string,
+    vendor?: Array<string>,
+    fonts?: Array<string>,
+    bundle: Array<string>,
+    css: string,
+    html: string,
+    data: {
+        page: string,
+        site: string,
+    },
+    cssIDs: Array<string>,
+    nonBlockingJS?: string,
+}) => {
+    const vendorPath = hashedPath(...vendor);
```

Comment:
this feels like it hsould be an array of already defined paths maybe? so the template just gets what it needs? might need to define critical (preloading) scripts, non-critical etc?

Reply:
Yep, I see what you're saying, if `htmlTemplate` is going to be generic enough to be used by multiple sites we should allow the sites to add as many scripts as they need. However as this PR doesn't introduce the problem of `htmlTemplate` being too opinionated about the scripts used I'd say I should raise this a new ticket on the `dotComponent` todo list. Does that sound OK?